src/Atmos/Parameterizations/CloudPhysics/Microphysics.jl

```diff
@@ -0,0 +1,187 @@
+"""
+    one-moment bulk Microphysics scheme
+
+Microphysics parameterization based on the ideas of Kessler_1995:
+  - condensation/evaporation as relaxation to equilibrium
+  - autoconversion
+  - accretion
+  - rain evaporation
+  - rain terminal velocity
+"""
+
+module Microphysics
+
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using ..MicrophysicsParameters
+
+# rain fall speed
+export terminal_velocity_single_drop_coeff
+export terminal_velocity
+
+# rates of conversion between microphysics categories
+export conv_q_vap_to_q_liq
+export conv_q_liq_to_q_rai_acnv
+export conv_q_liq_to_q_rai_accr
+export conv_q_rai_to_q_vap
+
+"""
+    terminal_velocity_single_drop_coeff(ρ)
+
+where:
+  - `ρ` is the density of air
+
+Returns the proportionality coefficient between terminal velocity of an
+individual water drop and the square root of its radius * g.
+"""
+function terminal_velocity_single_drop_coeff(ρ::DT) where {DT<:Real}
+
+    # terminal_vel_of_individual_drop = v_drop_coeff * (g * drop_radius)^(1/2)
+    v_c::DT = sqrt(DT(8/3) / C_drag * (dens_liquid / ρ - DT(1)))
```

Comment:
```suggestion
    return sqrt(DT(8/3) / C_drag * (dens_liquid / ρ - DT(1)))
```
and get rid of empty whitespace

Reply:
done